File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/largepicture/LargeImageInfo.java
package com.didichuxing.doraemonkit.kit.largepicture;

/**
 * ================================================
 * 作    者：jint（金台）
 * 版    本：1.0
 * 创建日期：2019-09-26-17:28
 * 描    述：大图信息
 * 修订历史：
 * ================================================
 */
public class LargeImageInfo {
    private String url;
    private String fileSize;
    private String memorySize;
    private int width;
    private int height;
    private String from;

    public void setUrl(String url) {
        this.url = url;
    }

    public String getFileSize() {
        return fileSize;
    }

    public void setFileSize(String fileSize) {
        this.fileSize = fileSize;
    }

    public void setMemorySize(String memorySize) {
        this.memorySize = memorySize;
    }

    public void setWidth(int width) {
        this.width = width;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public String getFrom() {
        return from;
    }

    public void setFrom(String from) {
        this.from = from;
    }

    public String getUrl() {
        return url;
    }

    public String getMemorySize() {
        return memorySize;
    }

    public int getWidth() {
        return width;
    }

    public int getHeight() {
        return height;
    }


    @Override
    public String toString() {
        return "LargeImageInfo{" +
                "url='" + url + '\'' +
                ", fileSize='" + fileSize + '\'' +
                ", memorySize='" + memorySize + '\'' +
                ", width=" + width +
                ", height=" + height +
                ", from='" + from + '\'' +
                '}';
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/largepicture/LargeImageInfo.kt
package com.didichuxing.doraemonkit.kit.largepicture

/**
 * @author:  maple
 * @time:  2020/6/8 - 14:01
 * @desc: 大图信息
 */
class LargeImageInfo {
    private var framework = "other"
    private var url: String? = null
    private var fileSize = 0.0
    private var memorySize = 0.0
    private var width = 0
    private var height = 0

    fun setUrl(url: String?) {
        this.url = url
    }

    fun getFileSize(): Double {
        return fileSize
    }

    fun setFileSize(fileSize: Double) {
        this.fileSize = fileSize
    }

    fun setMemorySize(memorySize: Double) {
        this.memorySize = memorySize
    }

    fun setWidth(width: Int) {
        this.width = width
    }

    fun setHeight(height: Int) {
        this.height = height
    }

    fun getUrl(): String? {
        return url
    }

    fun getMemorySize(): Double {
        return memorySize
    }

    fun getWidth(): Int {
        return width
    }

    fun getHeight(): Int {
        return height
    }


    fun getFramework(): String? {
        return framework
    }

    fun setFramework(framework: String) {
        this.framework = framework
    }

    override fun toString(): String {
        return "LargeImageInfo{" +
                "framework='" + framework + '\'' +
                ", url='" + url + '\'' +
                ", fileSize='" + fileSize + '\'' +
                ", memorySize='" + memorySize + '\'' +
                ", width=" + width +
                ", height=" + height +
                '}'
    }
}